### DISCUSSION TITLE
Add SCIM endpoints

### DISCUSSION THREAD

**Discussion by rijkvanzanten** [directus/directus#3113] [Feature Requests]

**Comment by w0kyj** [comment]
Some useful resource references:
https://datatracker.ietf.org/doc/html/rfc7644#section-3.2
https://developer.okta.com/docs/concepts/scim/

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
